src/orders/components/OrderListPage/OrderListPage.tsx

```diff
@@ -78,13 +86,38 @@ const OrderListPage: React.FC<OrderListPageProps> = ({
   const extensionMenuItems = mapToMenuItems(ORDER_OVERVIEW_MORE_ACTIONS);
   const extensionCreateButtonItems = mapToMenuItems(ORDER_OVERVIEW_CREATE);
 
+  const context = useDevModeContext();
+
+  const openPlaygroundURL = () => {
+    context.setDevModeContent(DevModeQuery);
+    const variables = JSON.stringify(
+      {
+        filter: getFilterVariables(params),
+        // TODO strange error when uncommenting this line
```

Comment:
**suggestion**: if there is a "todo" in the code, please create an issue for that and leave link to it here, within the comment, so you/we can track it